unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -14,18 +14,83 @@
     LanguageModelTool,
 )
 
+logger = logging.getLogger(f"toolkit.{__name__}")
 
-class LanguageModelService(BaseService):
+
+class LanguageModelService:
     """
     Provides methods to interact with the Language Model by generating responses.
 
     Attributes:
-        event (Event): The Event object.
-        logger (Optional[logging.Logger]): The logger object. Defaults to None.
+        company_id (str): The company ID.
+        user_id (Optional[str]): The user ID.
+        assistant_message_id (Optional[str]): The assistant message ID.
+        user_message_id (Optional[str]): The user message ID.
+        chat_id (Optional[str]): The chat ID.
+        assistant_id (Optional[str]): The assistant ID.
     """
 
-    def __init__(self, event: Event, logger: Optional[logging.Logger] = None):
```

Comment:
This will be a breaking change no? Perhaps better to make it an optional parameter?

Reply:
yes, a breaking change but I think easy to spot "LanguageModelService(event"

Reply:
uh great catch @sheiksadique. I think we could use [this](https://github.com/dabeaz/python-cookbook/blob/master/src/9/multiple_dispatch_with_function_annotations/example1.py) to avoid it. 

```
import time
class Date(metaclass=MultipleMeta):
    def __init__(self, year: int, month:int, day:int):
        self.year = year
        self.month = month
        self.day = day

    def __init__(self):
        t = time.localtime()
        self.__init__(t.tm_year, t.tm_mon, t.tm_mday)
```

Reply:
Not a big fan of eiterh. For now, I added a deprecated from_event method. It will be a bit of monkey work but I will make sure that there won't be any breaking changes.